src/views/topics/GdpTopic26View.vue

```diff
@@ -0,0 +1,384 @@
+<template>
```

Comment:
Is this tested?

Reply:
I will change the topic 26, but 25 and 27-32 are all test clean